<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statement Cruncher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton, QLabel {
	color: #323233;
	font-family: &quot;Source Code Pro Semibold&quot;;
}

QLabel {
	font-size: 24px;
}

QLabel[objectName=&quot;incomingsLabel&quot;],
QLabel[objectName=&quot;incomingsLabelPrice&quot;] {
	color: rgb(0, 170, 0);
}

QLabel[objectName=&quot;outgoingsLabel&quot;],
QLabel[objectName=&quot;outgoingsLabelPrice&quot;] {
	
	color: rgb(170, 0, 0);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="incomingsLabelPrice">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outgoingsLabelPrice">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sort Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Account</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debit Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credit Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Balance</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionPrint"/>
   <addaction name="actionAdd_Row"/>
   <addaction name="actionDelete_row"/>
   <addaction name="actionImport_Wizard"/>
   <addaction name="actionMege_Rows"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Wizard"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMege_Rows"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Row"/>
    <addaction name="actionDelete_row"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/new-document.png</normaloff>images/new-document.png</iconset>
   </property>
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="icon">
    <iconset>
     <normaloff>images/new-document.png</normaloff>images/new-document.png</iconset>
   </property>
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPage">
   <property name="icon">
    <iconset>
     <normaloff>images/printer.png</normaloff>images/printer.png</iconset>
   </property>
   <property name="text">
    <string>Page setup</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>images/printer.png</normaloff>images/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>images/undo.png</normaloff>images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>images/redo.png</normaloff>images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>images/cut.png</normaloff>images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>images/copy.png</normaloff>images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>images/paste.png</normaloff>images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset>
     <normaloff>images/find.png</normaloff>images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_next">
   <property name="icon">
    <iconset>
     <normaloff>images/find.png</normaloff>images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find next</string>
   </property>
  </action>
  <action name="actionFind_previous">
   <property name="icon">
    <iconset>
     <normaloff>images/find.png</normaloff>images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find previous</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset>
     <normaloff>images/replace.png</normaloff>images/replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionGo_to">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionDate_Time">
   <property name="text">
    <string>Date/Time</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset>
     <normaloff>images/font.png</normaloff>images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionStatus_bar">
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="actionWord_wrap">
   <property name="text">
    <string>Word wrap</string>
   </property>
  </action>
  <action name="actionAdd_Row">
   <property name="icon">
    <iconset>
     <normaloff>images/plus.png</normaloff>images/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Row</string>
   </property>
  </action>
  <action name="actionAdd_column">
   <property name="icon">
    <iconset>
     <normaloff>images/plus.png</normaloff>images/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add column</string>
   </property>
  </action>
  <action name="actionDelete_row">
   <property name="icon">
    <iconset>
     <normaloff>images/delete.png</normaloff>images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </action>
  <action name="actionDelete_column">
   <property name="icon">
    <iconset>
     <normaloff>images/delete.png</normaloff>images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete column</string>
   </property>
  </action>
  <action name="actionImport_Wizard">
   <property name="icon">
    <iconset>
     <normaloff>images/wizard.png</normaloff>images/wizard.png</iconset>
   </property>
   <property name="text">
    <string>Import Wizard</string>
   </property>
  </action>
  <action name="actionFont_Increase">
   <property name="icon">
    <iconset>
     <normaloff>images/font.png</normaloff>images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font Increase</string>
   </property>
  </action>
  <action name="actionFont_Decrease">
   <property name="icon">
    <iconset>
     <normaloff>images/font.png</normaloff>images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font Decrease</string>
   </property>
  </action>
  <action name="actionMege_Rows">
   <property name="icon">
    <iconset>
     <normaloff>images/merge.png</normaloff>images/merge.png</iconset>
   </property>
   <property name="text">
    <string>Merge Rows</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
